src/types/composer.ts

```diff
@@ -335,6 +444,7 @@ export default class AlgokitComposer {
       txn.fee = txn.estimateSize() * suggestedParams.fee || algosdk.ALGORAND_MIN_TX_FEE
       if (params.extraFee) txn.fee += params.extraFee.microAlgos
     }
+    txn.flatFee = true
```

Comment:
I assume this is needed unless I'm misunderstanding the fee control?

Reply:
Ah yep good call